proto/viam/app/v1/app.proto

```diff
@@ -1341,6 +1370,7 @@ message Uploads {
 message ListModulesRequest {
   // The id of the organization to return private modules for.
   optional string organization_id = 1;
+  optional bool include_markdown_documentation = 2;
```

Comment:
i don't think `ListModules` would ever want to return markdown, but does this have to do with this concern? 
> The alternative of the bool is to make separate messages for Request messages and Response messages. I tried that in previous commits in this PR. Its really gross and has backwards compatibility issues.

Reply:
Yeah this runs into the same issue that I was trying to fix by making a separate `Model` message. In order to not have a `markdown_documentation` field the response would have to be changed to a different message and cause backwards incompatibility. I could just remove this bool and always return a nil markdown. But I like the consistency of having the bool on all APIs.